### PR TITLE
chore: [#177204492] Update/Correct README

### PR DESCRIPTION
**Pull request by andreafavaro**
## Short description
Added new troubleshooting line to solve "bundle install" error

## How to test
After executing the command "bundle install" an error similar to this "Can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)" may come out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -900,6 +900,8 @@ $ detox test
 
 ### Troubleshooting
 
+If you get an error like this `Can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)` after launching `bundle install` you can fix it launching this `gem install bundler -v "$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)`
```
Comment (debiff) [inline README.md:903]: Can you add a subtitle to the section you added? 
For example: 
```suggestion
### Bundler install error
If you get an error like this `Can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)` after launching `bundle install` you can fix it launching this `gem install bundler -v "$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)`
```

**Review by debiff — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andreafavaro) [reply README.md:905]: done

**Comment by debiff** [comment]
Can you translate the title in english? This is our official language on Github.

**Comment by andreafavaro** [comment]
> @andreafavaro Can you translate the title in english? This is our official language on Github.

done

**Review by debiff — APPROVED** [review]

**Pull request merged**